src/main/resources/docker/engine.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
The spec expects this file at `/engine.json`. Could you add a `COPY` to the dockerfile?

```dockerfile
COPY engine.json /
```

Reply:
Done